web/app/themes/mitlib-child/functions.php

```diff
@@ -32,3 +32,127 @@ function child_scripts_styles() {
 	wp_enqueue_script( 'bootstrap-js' );
 }
 add_action( 'wp_enqueue_scripts', 'Mitlib\Child\child_scripts_styles' );
+
+/**
+ * Remove widget areas inherited from the Parent theme.
+ */
+function remove_parent_widgets() {
+	unregister_sidebar( 'sidebar-1' );
+	unregister_sidebar( 'sidebar-2' );
+	unregister_sidebar( 'sidebar-3' );
+}
+add_action( 'widgets_init', 'Mitlib\Child\remove_parent_widgets', 11 );
+
+/**
+ * Registers the Widgetized Area Below the Content.
+ */
+function sidebars_init() {
+	register_sidebar(
+		array(
+			'name' => __( 'Main Sidebar', 'twentytwelve' ),
+			'id' => 'sidebar',
+			'description' => __( 'Appears on posts and pages', 'twentytwelve' ),
+			'before_widget' => '<aside id="%1$s" class="widget %2$s" role="complementary">',
+			'after_widget' => '</aside>',
+			'before_title' => '<h2 class="widget-title">',
+			'after_title' => '</h2>',
+		)
+	);
+
+	register_sidebar(
+		array(
+			'name' => __( 'Below Content Widget Area', 'twentytwelve' ),
+			'id' => 'sidebar-two',
+			'description' => __( 'Appears when using the Front Page or Widgetized Page templates', 'twentytwelve' ),
+			'before_widget' => '<div id="%1$s" class="widget %2$s" role="complementary">',
+			'after_widget' => '</div>',
+			'before_title' => '<h3 class="widget-title">',
+			'after_title' => '</h3>',
+		)
+	);
+}
+add_action( 'widgets_init', 'Mitlib\Child\sidebars_init' );
+
+/**
+ * Define register_child_nav function.
+ */
+function register_child_nav() {
+	register_nav_menu( 'child-nav', 'Child Nav' );
+}
+add_action( 'init', 'Mitlib\Child\register_child_nav' );
+
+/**
+ * Remove parent theme page templates.
+ *
+ * The Parent theme includes a number of page templates which are meant for only
```

Comment:
Hmm. I wonder if this is a sign that they shouldn't be in the parent theme. I'm absolutely not suggesting we try to make a change now to our theme hierarchy but this feels like a code smell.

Reply:
Hmm... that's an interesting notion. I'd like to discuss this further. I agree that there's an aspect of code smell here, but the alternatives feel icky too.

Reply:
https://mitlibraries.atlassian.net/browse/LM-199 is now in the backlog to have this discussion